test/acceptance/.dockerignore

```diff
@@ -0,0 +1,7 @@
+node_modules
+reports
+test-output
+.dockerignore
```

Comment:
Why is .dockerignore in the .dockerignore?

Reply:
That's a good question.. refactored file

Reply:
It should be in there, you don't need to copy the .dockerignore file into the container

Reply:
Re-included